lib/bearerstrategy.js

```diff
@@ -293,8 +293,9 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
 
       var metadataURL = self._options.identityMetadata;
 
-      if (self._options.policyName) {
-        log.info('B2C: using policy %s', self._options.policyName);
+      if (!self._options.policyName) {
```

Comment:
Should we really be erroring any time there's no policyName? What if we're not using B2C- there'd be no policy here, right? And why remove the log info on the policy?